coalib/parsing/filters/CanDetectFilter.py

```diff
@@ -0,0 +1,14 @@
+from coalib.parsing.filters.Filter import filter
+
+
+@filter
+def can_detect_filter(bear, args):
+    """
+    Filters the bears by ``LANGUAGES``.
+
+    :param bear:        Bear object
+    :param args:        Set of detects on which bear is to be filtered
+    :return:            True if this bear is to be displayed for given
+                        args, False otherwise
+    """
+    return {detect.lower() for detect in bear.CAN_DETECT} & args
```

Comment:
not sure, maybe we should explicitly "boolify" the value?
```
return bool({detect.lower() for detect in bear.CAN_DETECT} & args)
```
I don't know. Up to you. There's actually no real reason to do this, except that it's maybe slightly better readable.